deployment/updates/scripts/2016_04_05_migrate_tvinci_distribution_tags_to_be_dynamic.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+$realrun = false;
+
+if($argc != 2)
+{
+	echo "Arguments missing.\n\n";
+	echo "Usage: php 2016_04_05_migrate_tvinci_distribution_tags_to_be_dynamic.php {partner id}\n";
+	exit;
+}
+require_once (__DIR__ . '/../../bootstrap.php');
+$partnerId = $argv[1];
+
+
+$c = new Criteria();
+$c->add(DistributionProfilePeer::PARTNER_ID, $partnerId);
+$c->add(DistributionProfilePeer::STATUS, DistributionProfileStatus::ENABLED);
+$c->add(DistributionProfilePeer::PROVIDER_TYPE, TvinciDistributionPlugin::getDistributionProviderTypeCoreValue(TvinciDistributionProviderType::TVINCI));
+$distributionProfiles = DistributionProfilePeer::doSelect($c, $con);
+
+foreach ($distributionProfiles as $distributionProfile)
+{
+	/**
+	 * @var TvinciDistributionProfile $distributionProfile
+	 */
+	$tags = array();
+	$ismFileName = $distributionProfile->getFromCustomData('ismFileName', null, '');
+	$ismPpvmodule = $distributionProfile->getFromCustomData('ismPpvModule', null, '');
+	if ( $ismFileName != '' || $ismPpvmodule != '' )
```

Comment:
since there was no delete to field there are file name and ppv module with space as empty , trim before comparing to be on the safe side